ijs.tree

```diff
@@ -232,6 +232,9 @@
     </toc-element>
   </toc-element>
   <toc-element toc-title="Testing">
+    <toc-element toc-title="Integration Tests">
```

Comment:
assuming this will be series of pages, there should be a landing page explaining this fact and linking the sub-pages with some more info

like https://plugins.jetbrains.com/docs/intellij/writing-tests-for-plugins.html